ports/duckdb/unvendor_icu_and_find_dependency.patch

```diff
@@ -8,13 +25,12 @@ index 65cbe38..a1f02f2 100644
  include_directories(include)
 -include_directories(third_party/icu/common)
 -include_directories(third_party/icu/i18n)
--
--add_subdirectory(third_party)
 +option(WITH_INTERNAL_ICU "Use vendored copy of icu" TRUE)
 +if(WITH_INTERNAL_ICU)
 +    include_directories(third_party/icu/common)
 +    include_directories(third_party/icu/i18n)
-+    
+ 
```

Comment:
This line changed because

```console
PS D:\vcpkg\buildtrees\duckdb\src\v1.2.0-9b06202205.clean> git apply D:\vcpkg\ports\duckdb\unvendor_icu.patch
D:/vcpkg/ports/duckdb/unvendor_icu.patch:17: trailing whitespace.

warning: 1 line adds whitespace errors.
```